test/VisualizationTests/VisualizationDataTests.jl

```diff
@@ -0,0 +1,31 @@
+module TestVisualizationData
+
+using Test
+using Gridap
+using Gridap.Visualization
+
+domain = (0,2pi, 0, 4pi)
+partition = (2, 3)
+model = simplexify(CartesianDiscreteModel(domain,partition))
+f(pt) = sin(pt[1])
+V0 = TestFESpace(
+  reffe=:Lagrangian, order=1, valuetype=Float64,
+  conformity=:H1, model=model)
+U = TrialFESpace(V0)
+trian = Triangulation(model)
+degree = 2
+quad = CellQuadrature(trian,degree)
+A(u, v) = u ⊙ v
+b(v) = v⊙f
+t_Ω = AffineFETerm(A,b,trian,quad)
+op = AffineFEOperator(U,V0,t_Ω)
+u = solve(op)
```

Comment:
What is a simpler way to create fields then solving a FEM problem?

Reply:
By interpolation
```julia
f(x) = sin(4*x[1])*cos(3*x[2])
u = interpolate(U,f)
```

In this case U can even be a test space without boundary conditions. You don't need to build a trial one.

Reply:
perfect thanks!

Reply:
Btw. it would be nice if the argument order of `interpolate` was switched. The usual convention is functions first, as it enables do block syntax.

Reply:
> Btw. it would be nice if the argument order of interpolate was switched. The usual convention is functions first, as it enables do block syntax.

true.